tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -332,7 +346,7 @@ def test_aot_executor(hexagon_session):
 
 
 @requires_hexagon_toolchain
-def test_aot_executor_multiple_conv2d(hexagon_session):
+def test_aot_executor_multiple_conv2d(hexagon_launcher, aot_target_kind):
```

Comment:
Can we instead support aot_target_kind and its consumption via `get_target_and_session` in the hexagon_session fixture? That way we can keep the simplified calling code so that the user doesn't need to think about starting a hexagon session.

Reply:
so AoT host targets are pytest parameters and they are not used for graph executor. If I add it in `hexagon_session` it will generate extra tests for graph executor which doesn't make sense. I added aot_target and aot_host_target as fixtures.
We could fix this by adding aot_hexagon_session fixture.